setup.py

```diff
@@ -12,6 +12,7 @@
 import glob
 
 from setuptools import setup, find_packages
+from osbs import __version__ as osbs_client_version
```

Comment:
It's bad form to import `osbs` from `setup.py`, since it actually loads the module prior to installing it.
If that file ever imports something from `requirements.txt`, this will cease to work.

If we do go down this approach, let's add a big fat comment to `osbs/__init__.py` to clarify this.